app/pulp/app/tasks/importer.py

```diff
@@ -15,3 +16,16 @@ def delete(repo_name, importer_name):
     :type  importer_name:   str
     """
     models.Importer.objects.filter(name=importer_name, repository__name=repo_name).delete()
+
+
+@shared_task(base=UserFacingTask)
+def sync(importer_name):
+    """
+    Call sync on the specified importer.
+
+    Args:
+        importer_name (basestring): unique name to specify the Importer
+    """
+    importer = models.Importer.objects.get(name=importer_name).cast()
```

Comment:
I just realized that this won't work, because https://github.com/pulp/pulp/blob/3.0-dev/app/pulp/app/models/repository.py#L76

Importer.name is not unique. The model's uniqueness is name + repo, not name alone.